packages/components/src/components/TagWithOverflow/TagsWithOverflow.scss

```diff
@@ -0,0 +1,47 @@
+/*
+Copyright 2025 The Tekton Authors
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+    http://www.apache.org/licenses/LICENSE-2.0
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+@use '@carbon/react/scss/config' as *;
+
+// styling links as tags
+// needed as some of the carbon link styles interfere with the tag styles
+.tkn--overflow-menu-container a,
+.tkn--overflow-menu-container button {
+  font-size: 12px;
+  color: #e0e0e0 !important;
```

Comment:
Can we make the selector more specific and avoid use of `!important` here?
Also, is there a colour token we could use instead of the hardcoded value?